assets/app/styles/_spacing.less

```diff
@@ -0,0 +1,36 @@
+// Spacing utility classes
+// creates:
+//  .pad-sm, .pad-md, .pad-lg, etc
+//  .mar-sm, .mar-md, .mar-lg, etc
+//  .pad-left-sm, .mar-left-sm, etc
+//
+@sizes: none 0px e('!important'), sm 4px e(''), md 8px e(''), lg 12px e(''), xl 16px e(''), xxl 32px e('');
```

Comment:
ok :smile:
Comments for that can prob move [to the PR against console_3.1 branch](https://github.com/jwforres/origin/pull/26), I nixed these helper classes here so we can debate about `px` and other evils for as long as we want.